src/main/java/com/google/step/snippet/servlets/SearchServlet.java

```diff
@@ -96,7 +106,6 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
      */
     String geeksLink = getLink(GEEKS_CSE_ID, query);
     if (geeksLink != null) {
```

Comment:
remove conditional, can added in a separate PR w/ the scraper.

Reply:
I think we're just keeping this null check consistent across all three sites to ensure that if the link from CSE is null, there's no need to even try creating a card by calling the respective scraping functions. At least afaik this is how Iris and I set up our clients.

Reply:
Sure, I just meant that because there is no scraper class yet, there's no reason to do the check rn.